editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -598,7 +598,9 @@ bool TileMapEditorTilesPlugin::forward_canvas_gui_input(const Ref<InputEvent> &p
 				}
 
 				if (drag_type == DRAG_TYPE_CLIPBOARD_PASTE) {
-					// Do nothing.
+					if (mb->get_button_index() == MouseButton::RIGHT) {
```

Comment:
Could you add a comment there? Something like "Cancel tile pasting on right-click"
This function is quite big, and such comment help to understand faster what each block of code is for.

Reply:
Done.